k8s/tools/check-lockfile.py

```diff
@@ -15,11 +15,14 @@ def main():
     timeout = values.get('timeout')
     if timeout > max_timeout:
       max_timeout = timeout
+  config.log.info(f'Set max timeout: {max_timeout}')
   try:
     lock = filelock.FileLock(config.LOCK_FILE)
     with lock.acquire(timeout=max_timeout):
+      config.log.info(f'Lock {config.LOCK_FILE} acquired')
```

Comment:
Do we want a log statement for when the lock is lifted too since we have one for when it is acquired?

Reply:
AFAICT we don't explicitly call lock.release() anywhere here so the only other place we can log is within the except block. Am i missing something?